.github/docker/Dockerfile.musl

```diff
@@ -0,0 +1,18 @@
+ARG PLATFORM=arm64
+ARG NODE_VERSION=16.20.1
+
+FROM ${PLATFORM}/node:${NODE_VERSION}-alpine AS build
```

Comment:
What version of musl does this use? Can we make a note of it in the readme alongside "Linux GLIBC 2.23"